### ISSUE TITLE
[Question] Equality constraints + non-negativity constraints

### ISSUE THREAD

**Issue by spring-haru** [avidale/constrained-linear-regression#9]
I wish to estimate

$y=A+Bx_1+Cx_2+u$

 subject to 

$B+C=1$, $B\geq 0$, $C\geq 0$

Can `MatrixConstrainedLinearRegression` do it? Does it handle inequality constrains only?

**Comment by avidale** [comment]
Yes, currently it handles only inequality constraints. But of course implementations of other types of tasks are welcome!

**Comment by spring-haru** [comment]
I decided to use `qpsolvers`. Thanks anyway.
